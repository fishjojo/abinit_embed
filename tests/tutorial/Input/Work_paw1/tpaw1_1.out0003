
.Version 8.10.2 of ABINIT 
.(MPI version, prepared for a x86_64_linux_intel18.0 computer) 

.Copyright (C) 1998-2018 ABINIT group . 
 ABINIT comes with ABSOLUTELY NO WARRANTY.
 It is free software, and you are welcome to redistribute it
 under certain conditions (GNU General Public License,
 see ~abinit/COPYING or http://www.gnu.org/copyleft/gpl.txt).

 ABINIT is a project of the Universite Catholique de Louvain,
 Corning Inc. and other collaborators, see ~abinit/doc/developers/contributors.txt .
 Please read https://docs.abinit.org/theory/acknowledgments for suggested
 acknowledgments of the ABINIT effort.
 For more information, see https://www.abinit.org .

.Starting date : Thu 18 Oct 2018.
- ( at 17h15 )
  
- input  file    -> tpaw1_1.in
- output file    -> tpaw1_1.out
- root for input  files -> tpaw1_1i
- root for output files -> tpaw1_1o
 
 
 Symmetries : space group Fd -3 m (#227); Bravais cF (face-center cubic)
================================================================================
 Values of the parameters that define the memory need of the present run
     intxc =       0    ionmov =       0      iscf =      17    lmnmax =       8
     lnmax =       4     mgfft =      18  mpssoang =       2    mqgrid =    3001
     natom =       2  nloc_mem =       2    nspden =       1   nspinor =       1
    nsppol =       1      nsym =      48    n1xccc =       1    ntypat =       1
    occopt =       1   xclevel =       1
-    mband =           6        mffmem =           1         mkmem =          28
       mpw =         219          nfft =        5832          nkpt =          28
 PAW method is used; the additional fine FFT grid is defined by:
   mgfftf=        32    nfftf =     32768
================================================================================
P This job should need less than                       7.473 Mbytes of memory.
  Rough estimation (10% accuracy) of disk space for files :
_ WF disk file :      0.563 Mbytes ; DEN or POT disk file :      0.252 Mbytes.
================================================================================
 
--------------------------------------------------------------------------------
------------- Echo of variables that govern the present computation ------------
--------------------------------------------------------------------------------
-
- outvars: echo of selected default values                                      
-   iomode0 =  0 , fftalg0 =312 , wfoptalg0 = 10
-
- outvars: echo of global parameters not present in the input file              
-  max_nthreads =    0
-
 -outvars: echo values of preprocessed input variables --------
            acell      6.7406531160E+00  6.7406531160E+00  6.7406531160E+00 Bohr
              amu      1.20110000E+01
             ecut      1.50000000E+01 Hartree
           ecutsm      5.00000000E-01 Hartree
-          fftalg         312
              ixc           7
              kpt     -8.33333333E-02 -1.66666667E-01  0.00000000E+00
                      -8.33333333E-02 -3.33333333E-01  0.00000000E+00
                      -1.66666667E-01 -2.50000000E-01  0.00000000E+00
                      -8.33333333E-02 -2.50000000E-01  8.33333333E-02
                      -8.33333333E-02  5.00000000E-01  0.00000000E+00
                      -1.66666667E-01 -4.16666667E-01  0.00000000E+00
                      -8.33333333E-02 -4.16666667E-01  8.33333333E-02
                      -2.50000000E-01 -3.33333333E-01  0.00000000E+00
                      -1.66666667E-01 -3.33333333E-01  8.33333333E-02
                      -8.33333333E-02 -3.33333333E-01  1.66666667E-01
                      -8.33333333E-02  3.33333333E-01  0.00000000E+00
                      -1.66666667E-01  4.16666667E-01  0.00000000E+00
                      -2.50000000E-01  5.00000000E-01  0.00000000E+00
                      -1.66666667E-01  5.00000000E-01  8.33333333E-02
                      -3.33333333E-01 -4.16666667E-01  0.00000000E+00
                      -2.50000000E-01 -4.16666667E-01  8.33333333E-02
                      -1.66666667E-01 -4.16666667E-01  1.66666667E-01
                      -8.33333333E-02 -4.16666667E-01  2.50000000E-01
                      -8.33333333E-02  1.66666667E-01  0.00000000E+00
                      -1.66666667E-01  2.50000000E-01  0.00000000E+00
                      -2.50000000E-01  3.33333333E-01  0.00000000E+00
                      -3.33333333E-01  4.16666667E-01  0.00000000E+00
                      -4.16666667E-01  5.00000000E-01  0.00000000E+00
                      -3.33333333E-01  5.00000000E-01  8.33333333E-02
                      -2.50000000E-01  5.00000000E-01  1.66666667E-01
                      -8.33333333E-02  0.00000000E+00  0.00000000E+00
                      -2.50000000E-01  0.00000000E+00  0.00000000E+00
                      -4.16666667E-01  0.00000000E+00  0.00000000E+00
         kptrlatt        6   -6    6     -6    6    6     -6   -6    6
          kptrlen      4.04439187E+01
P           mkmem          28
            natom           2
            nband           6
            ngfft          18      18      18
          ngfftdg          32      32      32
             nkpt          28
            nstep          20
             nsym          48
           ntypat           1
              occ      2.000000  2.000000  2.000000  2.000000  0.000000  0.000000
        pawecutdg      5.00000000E+01 Hartree
     plowan_bandi           1
     plowan_bandf           6
   plowan_compute           1
     plowan_natom           2
     plowan_iatom0       0    0
       plowan_nbl0       0    0
           prtden           0
           prteig           0
            prtwf           0
            rprim      0.0000000000E+00  5.0000000000E-01  5.0000000000E-01
                       5.0000000000E-01  0.0000000000E+00  5.0000000000E-01
                       5.0000000000E-01  5.0000000000E-01  0.0000000000E+00
           shiftk      5.00000000E-01  5.00000000E-01  5.00000000E-01
          spgroup         227
           symrel      1  0  0   0  1  0   0  0  1      -1  0  0   0 -1  0   0  0 -1
                       0 -1  1   0 -1  0   1 -1  0       0  1 -1   0  1  0  -1  1  0
                      -1  0  0  -1  0  1  -1  1  0       1  0  0   1  0 -1   1 -1  0
                       0  1 -1   1  0 -1   0  0 -1       0 -1  1  -1  0  1   0  0  1
                      -1  0  0  -1  1  0  -1  0  1       1  0  0   1 -1  0   1  0 -1
                       0 -1  1   1 -1  0   0 -1  0       0  1 -1  -1  1  0   0  1  0
                       1  0  0   0  0  1   0  1  0      -1  0  0   0  0 -1   0 -1  0
                       0  1 -1   0  0 -1   1  0 -1       0 -1  1   0  0  1  -1  0  1
                      -1  0  1  -1  1  0  -1  0  0       1  0 -1   1 -1  0   1  0  0
                       0 -1  0   1 -1  0   0 -1  1       0  1  0  -1  1  0   0  1 -1
                       1  0 -1   0  0 -1   0  1 -1      -1  0  1   0  0  1   0 -1  1
                       0  1  0   0  0  1   1  0  0       0 -1  0   0  0 -1  -1  0  0
                       1  0 -1   0  1 -1   0  0 -1      -1  0  1   0 -1  1   0  0  1
                       0 -1  0   0 -1  1   1 -1  0       0  1  0   0  1 -1  -1  1  0
                      -1  0  1  -1  0  0  -1  1  0       1  0 -1   1  0  0   1 -1  0
                       0  1  0   1  0  0   0  0  1       0 -1  0  -1  0  0   0  0 -1
                       0  0 -1   0  1 -1   1  0 -1       0  0  1   0 -1  1  -1  0  1
                       1 -1  0   0 -1  1   0 -1  0      -1  1  0   0  1 -1   0  1  0
                       0  0  1   1  0  0   0  1  0       0  0 -1  -1  0  0   0 -1  0
                      -1  1  0  -1  0  0  -1  0  1       1 -1  0   1  0  0   1  0 -1
                       0  0  1   0  1  0   1  0  0       0  0 -1   0 -1  0  -1  0  0
                       1 -1  0   0 -1  0   0 -1  1      -1  1  0   0  1  0   0  1 -1
                       0  0 -1   1  0 -1   0  1 -1       0  0  1  -1  0  1   0 -1  1
                      -1  1  0  -1  0  1  -1  0  0       1 -1  0   1  0 -1   1  0  0
            tnons      0.0000000  0.0000000  0.0000000     0.2500000  0.2500000  0.2500000
                       0.0000000  0.0000000  0.0000000     0.2500000  0.2500000  0.2500000
                       0.0000000  0.0000000  0.0000000     0.2500000  0.2500000  0.2500000
                       0.0000000  0.0000000  0.0000000     0.2500000  0.2500000  0.2500000
                       0.0000000  0.0000000  0.0000000     0.2500000  0.2500000  0.2500000
                       0.0000000  0.0000000  0.0000000     0.2500000  0.2500000  0.2500000
                       0.0000000  0.0000000  0.0000000     0.2500000  0.2500000  0.2500000
                       0.0000000  0.0000000  0.0000000     0.2500000  0.2500000  0.2500000
                       0.0000000  0.0000000  0.0000000     0.2500000  0.2500000  0.2500000
                       0.0000000  0.0000000  0.0000000     0.2500000  0.2500000  0.2500000
                       0.0000000  0.0000000  0.0000000     0.2500000  0.2500000  0.2500000
                       0.0000000  0.0000000  0.0000000     0.2500000  0.2500000  0.2500000
                       0.0000000  0.0000000  0.0000000     0.2500000  0.2500000  0.2500000
                       0.0000000  0.0000000  0.0000000     0.2500000  0.2500000  0.2500000
                       0.0000000  0.0000000  0.0000000     0.2500000  0.2500000  0.2500000
                       0.0000000  0.0000000  0.0000000     0.2500000  0.2500000  0.2500000
                       0.0000000  0.0000000  0.0000000     0.2500000  0.2500000  0.2500000
                       0.0000000  0.0000000  0.0000000     0.2500000  0.2500000  0.2500000
                       0.0000000  0.0000000  0.0000000     0.2500000  0.2500000  0.2500000
                       0.0000000  0.0000000  0.0000000     0.2500000  0.2500000  0.2500000
                       0.0000000  0.0000000  0.0000000     0.2500000  0.2500000  0.2500000
                       0.0000000  0.0000000  0.0000000     0.2500000  0.2500000  0.2500000
                       0.0000000  0.0000000  0.0000000     0.2500000  0.2500000  0.2500000
                       0.0000000  0.0000000  0.0000000     0.2500000  0.2500000  0.2500000
           tolvrs      1.00000000E-10
            typat      1  1
           useylm           1
              wtk        0.02778    0.02778    0.02778    0.05556    0.02778    0.02778
                         0.05556    0.02778    0.05556    0.05556    0.02778    0.02778
                         0.02778    0.05556    0.02778    0.05556    0.05556    0.05556
                         0.02778    0.02778    0.02778    0.02778    0.02778    0.05556
                         0.05556    0.00926    0.00926    0.00926
           xangst      0.0000000000E+00  0.0000000000E+00  0.0000000000E+00
                       8.9175000000E-01  8.9175000000E-01  8.9175000000E-01
            xcart      0.0000000000E+00  0.0000000000E+00  0.0000000000E+00
                       1.6851632790E+00  1.6851632790E+00  1.6851632790E+00
             xred      0.0000000000E+00  0.0000000000E+00  0.0000000000E+00
                       2.5000000000E-01  2.5000000000E-01  2.5000000000E-01
            znucl        6.00000
 
================================================================================
 
 chkinp: Checking input parameters for consistency.
 
 chkint_prt: ERROR -
  The value of the input variable plowan_iatom is 0, while it must be
  larger or equal to 1.
  Action: you should change the input variable plowan_iatom.
