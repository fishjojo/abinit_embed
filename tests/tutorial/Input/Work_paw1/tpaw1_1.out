
.Version 8.10.2 of ABINIT 
.(MPI version, prepared for a x86_64_linux_intel18.0 computer) 

.Copyright (C) 1998-2018 ABINIT group . 
 ABINIT comes with ABSOLUTELY NO WARRANTY.
 It is free software, and you are welcome to redistribute it
 under certain conditions (GNU General Public License,
 see ~abinit/COPYING or http://www.gnu.org/copyleft/gpl.txt).

 ABINIT is a project of the Universite Catholique de Louvain,
 Corning Inc. and other collaborators, see ~abinit/doc/developers/contributors.txt .
 Please read https://docs.abinit.org/theory/acknowledgments for suggested
 acknowledgments of the ABINIT effort.
 For more information, see https://www.abinit.org .

.Starting date : Thu 18 Oct 2018.
- ( at 18h17 )
  
- input  file    -> tpaw1_1.in
- output file    -> tpaw1_1.out
- root for input  files -> tpaw1_1i
- root for output files -> tpaw1_1o
 
 
 Symmetries : space group Fd -3 m (#227); Bravais cF (face-center cubic)
================================================================================
 Values of the parameters that define the memory need of the present run
     intxc =       0    ionmov =       0      iscf =      17    lmnmax =       8
     lnmax =       4     mgfft =      18  mpssoang =       2    mqgrid =    3001
     natom =       2  nloc_mem =       2    nspden =       1   nspinor =       1
    nsppol =       1      nsym =      48    n1xccc =       1    ntypat =       1
    occopt =       1   xclevel =       1
-    mband =           8        mffmem =           1         mkmem =          10
       mpw =         213          nfft =        5832          nkpt =          10
 PAW method is used; the additional fine FFT grid is defined by:
   mgfftf=        32    nfftf =     32768
================================================================================
P This job should need less than                       7.002 Mbytes of memory.
  Rough estimation (10% accuracy) of disk space for files :
_ WF disk file :      0.262 Mbytes ; DEN or POT disk file :      0.252 Mbytes.
================================================================================
 
--------------------------------------------------------------------------------
------------- Echo of variables that govern the present computation ------------
--------------------------------------------------------------------------------
-
- outvars: echo of selected default values                                      
-   iomode0 =  0 , fftalg0 =312 , wfoptalg0 = 10
-
- outvars: echo of global parameters not present in the input file              
-  max_nthreads =    0
-
 -outvars: echo values of preprocessed input variables --------
            acell      6.7406531160E+00  6.7406531160E+00  6.7406531160E+00 Bohr
              amu      1.20110000E+01
             ecut      1.50000000E+01 Hartree
           ecutsm      5.00000000E-01 Hartree
-          fftalg         312
              ixc           7
              kpt     -1.25000000E-01 -2.50000000E-01  0.00000000E+00
                      -1.25000000E-01  5.00000000E-01  0.00000000E+00
                      -2.50000000E-01 -3.75000000E-01  0.00000000E+00
                      -1.25000000E-01 -3.75000000E-01  1.25000000E-01
                      -1.25000000E-01  2.50000000E-01  0.00000000E+00
                      -2.50000000E-01  3.75000000E-01  0.00000000E+00
                      -3.75000000E-01  5.00000000E-01  0.00000000E+00
                      -2.50000000E-01  5.00000000E-01  1.25000000E-01
                      -1.25000000E-01  0.00000000E+00  0.00000000E+00
                      -3.75000000E-01  0.00000000E+00  0.00000000E+00
         kptrlatt        4   -4    4     -4    4    4     -4   -4    4
          kptrlen      2.69626125E+01
P           mkmem          10
            natom           2
            nband           8
            ngfft          18      18      18
          ngfftdg          32      32      32
             nkpt          10
            nstep          20
             nsym          48
           ntypat           1
              occ      2.000000  2.000000  2.000000  2.000000  0.000000  0.000000
                       0.000000  0.000000
        pawecutdg      5.00000000E+01 Hartree
        pawprtvol           3
         pawprtwf           1
     plowan_bandi           1
     plowan_bandf           8
   plowan_compute           1
     plowan_natom           2
     plowan_iatom0       1    2
       plowan_nbl0       2    2
     plowan_lcalc0       0    1    0    1
  plowan_projcalc0       1    2    3    4
           prtvol           4
            rprim      0.0000000000E+00  5.0000000000E-01  5.0000000000E-01
                       5.0000000000E-01  0.0000000000E+00  5.0000000000E-01
                       5.0000000000E-01  5.0000000000E-01  0.0000000000E+00
           shiftk      5.00000000E-01  5.00000000E-01  5.00000000E-01
          spgroup         227
           symrel      1  0  0   0  1  0   0  0  1      -1  0  0   0 -1  0   0  0 -1
                       0 -1  1   0 -1  0   1 -1  0       0  1 -1   0  1  0  -1  1  0
                      -1  0  0  -1  0  1  -1  1  0       1  0  0   1  0 -1   1 -1  0
                       0  1 -1   1  0 -1   0  0 -1       0 -1  1  -1  0  1   0  0  1
                      -1  0  0  -1  1  0  -1  0  1       1  0  0   1 -1  0   1  0 -1
                       0 -1  1   1 -1  0   0 -1  0       0  1 -1  -1  1  0   0  1  0
                       1  0  0   0  0  1   0  1  0      -1  0  0   0  0 -1   0 -1  0
                       0  1 -1   0  0 -1   1  0 -1       0 -1  1   0  0  1  -1  0  1
                      -1  0  1  -1  1  0  -1  0  0       1  0 -1   1 -1  0   1  0  0
                       0 -1  0   1 -1  0   0 -1  1       0  1  0  -1  1  0   0  1 -1
                       1  0 -1   0  0 -1   0  1 -1      -1  0  1   0  0  1   0 -1  1
                       0  1  0   0  0  1   1  0  0       0 -1  0   0  0 -1  -1  0  0
                       1  0 -1   0  1 -1   0  0 -1      -1  0  1   0 -1  1   0  0  1
                       0 -1  0   0 -1  1   1 -1  0       0  1  0   0  1 -1  -1  1  0
                      -1  0  1  -1  0  0  -1  1  0       1  0 -1   1  0  0   1 -1  0
                       0  1  0   1  0  0   0  0  1       0 -1  0  -1  0  0   0  0 -1
                       0  0 -1   0  1 -1   1  0 -1       0  0  1   0 -1  1  -1  0  1
                       1 -1  0   0 -1  1   0 -1  0      -1  1  0   0  1 -1   0  1  0
                       0  0  1   1  0  0   0  1  0       0  0 -1  -1  0  0   0 -1  0
                      -1  1  0  -1  0  0  -1  0  1       1 -1  0   1  0  0   1  0 -1
                       0  0  1   0  1  0   1  0  0       0  0 -1   0 -1  0  -1  0  0
                       1 -1  0   0 -1  0   0 -1  1      -1  1  0   0  1  0   0  1 -1
                       0  0 -1   1  0 -1   0  1 -1       0  0  1  -1  0  1   0 -1  1
                      -1  1  0  -1  0  1  -1  0  0       1 -1  0   1  0 -1   1  0  0
            tnons      0.0000000  0.0000000  0.0000000     0.2500000  0.2500000  0.2500000
                       0.0000000  0.0000000  0.0000000     0.2500000  0.2500000  0.2500000
                       0.0000000  0.0000000  0.0000000     0.2500000  0.2500000  0.2500000
                       0.0000000  0.0000000  0.0000000     0.2500000  0.2500000  0.2500000
                       0.0000000  0.0000000  0.0000000     0.2500000  0.2500000  0.2500000
                       0.0000000  0.0000000  0.0000000     0.2500000  0.2500000  0.2500000
                       0.0000000  0.0000000  0.0000000     0.2500000  0.2500000  0.2500000
                       0.0000000  0.0000000  0.0000000     0.2500000  0.2500000  0.2500000
                       0.0000000  0.0000000  0.0000000     0.2500000  0.2500000  0.2500000
                       0.0000000  0.0000000  0.0000000     0.2500000  0.2500000  0.2500000
                       0.0000000  0.0000000  0.0000000     0.2500000  0.2500000  0.2500000
                       0.0000000  0.0000000  0.0000000     0.2500000  0.2500000  0.2500000
                       0.0000000  0.0000000  0.0000000     0.2500000  0.2500000  0.2500000
                       0.0000000  0.0000000  0.0000000     0.2500000  0.2500000  0.2500000
                       0.0000000  0.0000000  0.0000000     0.2500000  0.2500000  0.2500000
                       0.0000000  0.0000000  0.0000000     0.2500000  0.2500000  0.2500000
                       0.0000000  0.0000000  0.0000000     0.2500000  0.2500000  0.2500000
                       0.0000000  0.0000000  0.0000000     0.2500000  0.2500000  0.2500000
                       0.0000000  0.0000000  0.0000000     0.2500000  0.2500000  0.2500000
                       0.0000000  0.0000000  0.0000000     0.2500000  0.2500000  0.2500000
                       0.0000000  0.0000000  0.0000000     0.2500000  0.2500000  0.2500000
                       0.0000000  0.0000000  0.0000000     0.2500000  0.2500000  0.2500000
                       0.0000000  0.0000000  0.0000000     0.2500000  0.2500000  0.2500000
                       0.0000000  0.0000000  0.0000000     0.2500000  0.2500000  0.2500000
           tolvrs      1.00000000E-10
            typat      1  1
           useylm           1
              wtk        0.09375    0.09375    0.09375    0.18750    0.09375    0.09375
                         0.09375    0.18750    0.03125    0.03125
           xangst      0.0000000000E+00  0.0000000000E+00  0.0000000000E+00
                       8.9175000000E-01  8.9175000000E-01  8.9175000000E-01
            xcart      0.0000000000E+00  0.0000000000E+00  0.0000000000E+00
                       1.6851632790E+00  1.6851632790E+00  1.6851632790E+00
             xred      0.0000000000E+00  0.0000000000E+00  0.0000000000E+00
                       2.5000000000E-01  2.5000000000E-01  2.5000000000E-01
            znucl        6.00000
 
================================================================================
 
 chkinp: Checking input parameters for consistency.
 
================================================================================
== DATASET  1 ==================================================================
-   nproc =    1
 
 Exchange-correlation functional for the present dataset will be:
  LDA: Perdew-Wang 92 LSD fit to Ceperley-Alder data - ixc=7
 Citation for XC functional:
  J.P.Perdew and Y.Wang, PRB 45, 13244 (1992)
 
 Real(R)+Recip(G) space primitive vectors, cartesian coordinates (Bohr,Bohr^-1):
 R(1)=  0.0000000  3.3703266  3.3703266  G(1)= -0.1483536  0.1483536  0.1483536
 R(2)=  3.3703266  0.0000000  3.3703266  G(2)=  0.1483536 -0.1483536  0.1483536
 R(3)=  3.3703266  3.3703266  0.0000000  G(3)=  0.1483536  0.1483536 -0.1483536
 Unit cell volume ucvol=  7.6567760E+01 bohr^3
 Angles (23,13,12)=  6.00000000E+01  6.00000000E+01  6.00000000E+01 degrees
 
 Coarse grid specifications (used for wave-functions):
 
 getcut: wavevector=  0.0000  0.0000  0.0000  ngfft=  18  18  18
         ecut(hartree)=     15.000   => boxcut(ratio)=   2.17276
 
 Fine grid specifications (used for densities):
 
 getcut: wavevector=  0.0000  0.0000  0.0000  ngfft=  32  32  32
         ecut(hartree)=     50.000   => boxcut(ratio)=   2.10918
 
--- Pseudopotential description ------------------------------------------------
- pspini: atom type   1  psp file is ../../../Psps_for_tests/6c.lda.atompaw
- pspatm: opening atomic psp file    ../../../Psps_for_tests/6c.lda.atompaw
- Paw atomic data for element C - Generated by AtomPAW + AtomPAW2Abinit v3.2.1
-  6.00000   4.00000  20080326                znucl, zion, pspdat
    7    7    1    0       505   0.00000      pspcod,pspxc,lmax,lloc,mmax,r2well
 Pseudopotential format is: paw4
 basis_size (lnmax)=  4 (lmn_size=  8), orbitals=   0   0   1   1
 Spheres core radius: rc_sph= 1.50000000
 4 radial meshes are used:
  - mesh 1: r(i)=AA*[exp(BB*(i-1))-1], size= 505 , AA= 0.21824E-02 BB= 0.13095E-01
  - mesh 2: r(i)=AA*[exp(BB*(i-1))-1], size= 500 , AA= 0.21824E-02 BB= 0.13095E-01
  - mesh 3: r(i)=AA*[exp(BB*(i-1))-1], size= 530 , AA= 0.21824E-02 BB= 0.13095E-01
  - mesh 4: r(i)=AA*[exp(BB*(i-1))-1], size= 644 , AA= 0.21824E-02 BB= 0.13095E-01
 Shapefunction is SIN type: shapef(r)=[sin(pi*r/rshp)/(pi*r/rshp)]**2
 Radius for shape functions = sphere core radius
 Radial grid used for partial waves is grid 1
 Radial grid used for projectors is grid  2
 Radial grid used for (t)core density is grid 3
 Radial grid used for Vloc is grid 4
 Radial grid used for pseudo valence density is grid 4
 Compensation charge density is taken into account in XC energy/potential
 pspatm: atomic psp has been read  and splines computed
 
   4.14245303E+01                                ecore*ucvol(ha*bohr**3)
--------------------------------------------------------------------------------
 
P newkpt: treating      8 bands with npw=     209 for ikpt=   1 by node    0
P newkpt: treating      8 bands with npw=     213 for ikpt=   2 by node    0
P newkpt: treating      8 bands with npw=     213 for ikpt=   3 by node    0
P newkpt: treating      8 bands with npw=     211 for ikpt=   4 by node    0
P newkpt: treating      8 bands with npw=     212 for ikpt=   5 by node    0
P newkpt: treating      8 bands with npw=     212 for ikpt=   6 by node    0
P newkpt: treating      8 bands with npw=     211 for ikpt=   7 by node    0
P newkpt: treating      8 bands with npw=     212 for ikpt=   8 by node    0
P newkpt: treating      8 bands with npw=     202 for ikpt=   9 by node    0
P newkpt: treating      8 bands with npw=     211 for ikpt=  10 by node    0
_setup2: Arith. and geom. avg. npw (full set) are     211.281     211.272
 
================================================================================
 
     iter   Etot(hartree)      deltaE(h)  residm     nres2
 ETOT  1  -11.534954443117    -1.153E+01 2.042E-01 1.423E+01
 ETOT  2  -11.528011461994     6.943E-03 9.848E-05 1.373E+00
 ETOT  3  -11.525673869309     2.338E-03 6.510E-05 2.412E-02
 ETOT  4  -11.525680338015    -6.469E-06 2.967E-06 5.876E-04
 ETOT  5  -11.525680408963    -7.095E-08 4.602E-07 2.187E-05
 ETOT  6  -11.525680423537    -1.457E-08 3.035E-08 4.176E-08
 ETOT  7  -11.525680423600    -6.356E-11 3.635E-09 1.707E-09
 ETOT  8  -11.525680423600    -1.616E-13 2.524E-10 2.920E-12
 
 At SCF step    8       nres2   =  2.92E-12 < tolvrs=  1.00E-10 =>converged.
 
 Cartesian components of stress tensor (hartree/bohr^3)
  sigma(1 1)=  4.20916693E-04  sigma(3 2)=  0.00000000E+00
  sigma(2 2)=  4.20916693E-04  sigma(3 1)=  0.00000000E+00
  sigma(3 3)=  4.20916693E-04  sigma(2 1)=  0.00000000E+00
 
 Integrated electronic density in atomic spheres:
 ------------------------------------------------
 Atom  Sphere_radius  Integrated_density
    1        1.50000          2.63132455
    2        1.50000          2.63132455
 
 PAW TEST:
 ==== Compensation charge inside spheres ============
 The following values must be close to each other ...
 Compensation charge over spherical meshes =      0.413221948184052
 Compensation charge over fine fft grid    =      0.413220372269609
 
 ==== Results concerning PAW augmentation regions ====
 
 Total pseudopotential strength Dij (hartree):
 Atom #  1
   0.59010  -1.64886   0.00000   0.00000   0.00000   0.00000   0.00000   0.00000
  -1.64886   4.11691   0.00000   0.00000   0.00000   0.00000   0.00000   0.00000
   0.00000   0.00000  -0.03415   0.00000   0.00000  -0.22979   0.00000   0.00000
   0.00000   0.00000   0.00000  -0.03415   0.00000   0.00000  -0.22979   0.00000
   0.00000   0.00000   0.00000   0.00000  -0.03415   0.00000   0.00000  -0.22979
   0.00000   0.00000  -0.22979   0.00000   0.00000   2.94469   0.00000   0.00000
   0.00000   0.00000   0.00000  -0.22979   0.00000   0.00000   2.94469   0.00000
   0.00000   0.00000   0.00000   0.00000  -0.22979   0.00000   0.00000   2.94469
 Atom #  2
   0.59010  -1.64886   0.00000   0.00000   0.00000   0.00000   0.00000   0.00000
  -1.64886   4.11691   0.00000   0.00000   0.00000   0.00000   0.00000   0.00000
   0.00000   0.00000  -0.03415   0.00000   0.00000  -0.22979   0.00000   0.00000
   0.00000   0.00000   0.00000  -0.03415   0.00000   0.00000  -0.22979   0.00000
   0.00000   0.00000   0.00000   0.00000  -0.03415   0.00000   0.00000  -0.22979
   0.00000   0.00000  -0.22979   0.00000   0.00000   2.94469   0.00000   0.00000
   0.00000   0.00000   0.00000  -0.22979   0.00000   0.00000   2.94469   0.00000
   0.00000   0.00000   0.00000   0.00000  -0.22979   0.00000   0.00000   2.94469
 
 
 Augmentation waves occupancies Rhoij:
 Atom #  1
   1.61821   0.04513   0.00000   0.00000   0.00000   0.00000   0.00000   0.00000
   0.04513   0.00173   0.00000   0.00000   0.00000   0.00000   0.00000   0.00000
   0.00000   0.00000   1.40945   0.00000   0.00000   0.06176   0.00000   0.00000
   0.00000   0.00000   0.00000   1.40945   0.00000   0.00000   0.06176   0.00000
   0.00000   0.00000   0.00000   0.00000   1.40945   0.00000   0.00000   0.06176
   0.00000   0.00000   0.06176   0.00000   0.00000   0.00320   0.00000   0.00000
   0.00000   0.00000   0.00000   0.06176   0.00000   0.00000   0.00320   0.00000
   0.00000   0.00000   0.00000   0.00000   0.06176   0.00000   0.00000   0.00320
 Atom #  2
   1.61821   0.04513   0.00000   0.00000   0.00000   0.00000   0.00000   0.00000
   0.04513   0.00173   0.00000   0.00000   0.00000   0.00000   0.00000   0.00000
   0.00000   0.00000   1.40945   0.00000   0.00000   0.06176   0.00000   0.00000
   0.00000   0.00000   0.00000   1.40945   0.00000   0.00000   0.06176   0.00000
   0.00000   0.00000   0.00000   0.00000   1.40945   0.00000   0.00000   0.06176
   0.00000   0.00000   0.06176   0.00000   0.00000   0.00320   0.00000   0.00000
   0.00000   0.00000   0.00000   0.06176   0.00000   0.00000   0.00320   0.00000
   0.00000   0.00000   0.00000   0.00000   0.06176   0.00000   0.00000   0.00320
 
 
 
 ======================================================================================
 
 == Start computation of Projected Local Orbitals Wannier functions ==   0
 
 == Lower and upper values of the selected bands    1    8
 == Number of atoms                                      2
 == Atoms selected                                1 2
 == Nb of angular momenta used for each atom      2 2
 == Value of the angular momenta for atom 1 is :  0 1
 == Value of the projectors      for atom 1 is :  1 2
 == Value of the angular momenta for atom 2 is :  0 1
 == Value of the projectors      for atom 2 is :  3 4
